hr_employee_social_media/models/hr_social_media.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 Denis Leemann, Camptocamp SA
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class HrSocialMedia(models.Model):
+    _name = 'hr.social.media'
+    _description = 'HR Social Media'
+
+    name = fields.Char(
+        string='Name',
+    )
+    social_url = fields.Char(
+        string='Website',
+    )
+
+
+class HrSocialMediaAccount(models.Model):
+    _name = 'hr.social.media.account'
+    _description = 'HR Employee Social Media Account'
+
+    hr_social_media_id = fields.Many2one(
+        'hr.social.media',
+        string='Social Media',
+        required=True,
+    )
+    account_name = fields.Char(
```

Comment:
What do you think about changing the field name to `name`, to avoid this:

![captura de pantalla 2018-11-09 a la s 12 33 04](https://user-images.githubusercontent.com/7606656/48281278-b2370400-e41b-11e8-8f00-42d0eb4ea7a4.png)

Reply:
Added a `_get_name`